locales/ko/translation.json

```
@@ -0,0 +1,271 @@
+{
+    "$Language": {
+        "en": "English",
+        "pt-BR": "Português - Brasil",
+        "es": "Español",
+        "it": "Italiano",
+        "zh-TW": "繁體中文",
+        "de-DE": "Deutsch",
+        "hi": "हिंदी",
+        "mr": "मराठी",
+        "pl": "Polski",
+        "ko": "한국어"
+    },
+    "$Preferences": {
+        "title": "Time to Leave - 설정",
+        "userPreferences": "사용자 설정",
+        "workingDays": "작업 요일 지정",
+        "sun": "일요일",
+        "mon": "월요일",
+        "tue": "화요일",
+        "wed": "수요일",
+        "thu": "목요일",
+        "fri": "금요일",
+        "sat": "토요일",
```

In `DateUtil` you use less "characters" (sorry I don't know the name for the symbols). I think it could be used here too?

```suggestion
        "sun": "일",
        "mon": "월",
        "tue": "화",
        "wed": "수",
        "thu": "목",
        "fri": "금",
        "sat": "토",
```

---

Ohh you're right. That's correct!

---

Thanks!